code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm

```diff
@@ -28,6 +28,33 @@
 		light.turn_light(null, FALSE, duration, TRUE, TRUE)
 
 
+/datum/action/xeno_action/activable/nightfall/lesser //Less griefy lightfall.
+	name = "Lesser Nightfall"
+	action_icon_state = "nightfall"
+	ability_name = "Lesser Nightfall"
+	mechanics_text = "Shut down all electrical lights for five seconds."
+	cooldown_timer = 5 SECONDS //Nightfall buff
+	plasma_cost = 50 // Cheap
+	/// How far nightfall will have an effect
+	var/range = 8 //Buffed range
+	/// How long till the lights go on again
+	var/duration = 3 SECONDS // 2 Seconds of downtime, enough to make lights consistently flicker
+
+/datum/action/xeno_action/activable/nightfall/lesser/on_cooldown_finish()
+	to_chat(owner, "<span class='notice'>We gather enough mental strength to flicker lights again.</span>")
+	return ..()
+
+/datum/action/xeno_action/activable/nightfall/lesser/use_ability()
+	playsound(owner, 'sound/magic/nightfall.ogg', 50, 1)
+	succeed_activate()
+	add_cooldown()
+	for(var/atom/light AS in GLOB.nightfall_toggleable_lights)
+		if(isnull(light.loc) || (owner.loc.z != light.loc.z) || (get_dist(owner, light) >= range))
+			continue
+		light.turn_light(null, FALSE, duration, TRUE, TRUE)
```

Comment:
You don't actually need any of this re-defined here, since it's already on the parent datum `/datum/action/xeno_action/activable/nightfall`.
The `on_cooldown_finish()` one will actually display the notice twice, since it's calling the parent proc at the end.

Reply:
Also true, already fixed locally; this was just getting it up for review